docs/components/multiselect.md

```diff
@@ -287,26 +308,28 @@ See [autosuggest](#autosuggest-with-debounce) for a functional example.
 <KMultiselect loading :items="items" />
 ```
 
-### filterFunc
+### filterFunction
 
 Use this prop to override the default filter function if you want to do something like filter on an attribute other than `label`. Your filter function
-should take as parameter a JSON object containing the items to filter on (`items`) and the query string (`query`) and return the filtered items. See [Slots](#slots) for an example.
+should take as parameter a JSON object containing the items to filter on (`items`) and the query string (`query`) and return the filtered items. See [slots](#slots) for an example.
 
-```js
-myCustomFilter ({ items, query }) {
+```ts
+import { MultiselectFilterFnParams } from '@kong/kongponents'
+
+const myCustomFilter = ({ items, query }: MultiselectFilterFnParams) => {
   return items.filter(anItem => anItem.label.includes(query))
 }
 ```
 
 ::: tip NOTE
-`filterFunc` does not work with `autosuggest` enabled.
+`filterFunction` does not work with `autosuggest` enabled.
 For `autosuggest`, you are in charge of filtering the options, so `KMultiselect` won't filter them internally.
 See [autosuggest](#autosuggest) for more details.
 :::
 
 ### v-model
 
-`KMultiselect` works as regular inputs do using v-model for data binding:
+KMultiselect works as regular inputs do using v-model for data binding:
```

Comment:
Should we move the `v-model` prop docs under the `items` prop at the top since they are typically used together?